.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "shell",
+			"command": "(rm -rf ./deployments/hardhat) && (source .env) && (cp -r ./deployments/mainnet ./deployments/hardhat)",
```

Comment:
I split mine into copy and remove. eg

```js
        {
            "type": "shell",
            "command": "cp -r ./deployments/mainnet ./deployments/hardhat",
            "label": "copy-deployment-files",
            "options": {
                "cwd": "${workspaceFolder}/contracts"
            }
        },
        {
            "type": "shell",
            "command": "rm -rf ./deployments/hardhat",
            "label": "remove-deployment-files",
            "options": {
                "cwd": "${workspaceFolder}/contracts"
            }
        }   
```

The launch config can then have `"postDebugTask": "remove-deployment-files",`

Reply:
Good point. Can do the post action for removing, however if for some reason, the script/debugger/IDE crashes, it may not run the `postDebugTask`. Will have to manually delete the files and it can take a bit of time for someone to realize this and actually delete the files

Reply:
Good point.